src/CollapsibleField.js

```diff
@@ -319,9 +319,13 @@ class CollapsibleField extends Component {
     //remove header elements from the schema
     let headerElementsSchemas = {};
     let propertiesNoHeader = { ...properties };
-    let orderNoHeader = uiSchema["ui:order"]
-      ? uiSchema["ui:order"].filter(x => elements.indexOf(x) < 0)
-      : uiSchema["ui:order"];
+    let orderNoHeader = uiSchema["ui:order"];
+
+    if (orderNoHeader && orderNoHeader.filter) {
```

Comment:
What does `orderNoHeader.filter` do here?

Reply:
Array.filter.   ui:order can be either a single string "test" or an array of strings.  Checks which type it is.